rancilio-pid/rancilio-pid.ino

```diff
@@ -1043,6 +1189,21 @@ void updateState() {
       }
       break;
     }
+    case 6: //state 6 generating steam
+    {
+      bPID.SetAutoTune(false);  //do not tune during steam phase
+      bPID.SetSumOutputI(100);
+
+      if (activeSwitch != 3 && Input <= (setPoint + 1)) {
```

Comment:
Brauchen wir das denn? Wenn der steam Knopf "aus" ist, kann man doch sofort aus diesem "steam modus (activeState==6) gehen und in den activeSTATE=3 (normal modus) wechseln. 
Wenn man das so macht, dann haette man den die Steaming Phase sauber in der State-Maschine abgebildet und das cooldown ist dann in "Normal-Modus" bzw. "Outer-Zone". Das Display würde dann auch automatisch alles korrekt anzeigen. PS: Ich habe den Kd=0 Code in Outer-Zone bewegt, somit sollte sich das mMn identisch verhalten.

Spricht was dagegen?

Reply:
ja, war tatsächlich nur damit die Maschine beim Abkühlen wenn sie oberhalb des setpoints ist nichtnachheizt. Ggf wäre auch eine weitere outerZone oberhalb des setpoints sinnvoll, da die Maschinen sich ja unterschiedliche Heizraten bzw. Abkühlraten haben.